features/async_processes.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Asynchronous processes
+  The CCA DIP access application will use an asynchronous worker to execute long
+  running processes in the background. This worker will be implemented using Celery
+  and it will be initially used to extract and parse the METS file in the Folder
+  creation process after the DIP zip file is uploaded.
+
+  Scenario: Folder creation
+    Given an instance running with the asynchronous worker setup
+    When a user access the interface
+    And creates a new Folder attached to a Collection and with a DIP ZIP file
```

Comment:
do you mean `creates a new Folder with a DIP zip file that is attached to a collection` ?